test/kuttl/test-servicemonitors/03-assert.yaml

```
@@ -3,6 +3,22 @@ kind: StatefulSet
 metadata:
   name: test-dc1-default-sts
   namespace: k8ssandra-operator
+spec:
+  template:
+    spec:
+      containers:
+        - name: cassandra
+          env:
+            - name: MGMT_API_DISABLE_MCAC
+              value: "true"
+            - name: LOCAL_JMX
+            - name: METRIC_FILTERS
+            - name: DS_LICENSE
+            - name: DSE_AUTO_CONF_OFF
+            - name: USE_MGMT_API
+            - name: MGMT_API_EXPLICIT_START
+            - name: DSE_MGMT_EXPLICIT_START
+        - name: server-system-loggerßß
```

```suggestion
        - name: server-system-logger
```

Not sure if that was intentional :)

---

It wasn't. Thanks...